CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-## Unreleased
+## 2.3.0 [in progress]
+### Features
+ - [#264](https://github.com/influxdata/influxdb-client-go/pull/264) Synced generated server API with the latest [oss.yml](https://github.com/influxdata/openapi/blob/master/contracts/oss.yml). 
+
```

Comment:
This PR introduces breaking changes in domain model. What do you think about describing these changes in `CHANGELOG.md`?

Reply:
I did a couple of workarounds to avoid introducing breaking changes in the library API due to the changes in the server swagger and in the generated code. 
I don't think it is necessary to describe swagger changes that are outside of the written API.

Reply:
OK